### DISCUSSION TITLE
Domain with spanish letter "ñ"

### DISCUSSION THREAD

**Discussion by mmassetti** [vercel/vercel#4472] [Help] [answered]
Hi guys!
I have a question. Anyone knows if there is support or future plans for the posibility to have a domain containing the spanish letter "ñ" ? Thanks!

**Comment by mglagola** [comment, marked as answer]
Hey @mmassetti  - we support `punycode` domains.

For example: `café.com` can be added as `xn--caf-dma.com`.

Check out [punycoder.com](punycoder.com) if you need help converting your desired name to punycode.

**Reply by mmassetti** [reply]
Thanks Mark! Appreciate it.
